clientd3d/game.c

```diff
@@ -687,12 +684,12 @@ player_info *GetPlayerInfo(void)
    return &player;
 }
 
-void SetActiveGroup(BYTE group)
-{
-    active_group = group;
+void SetActiveStatGroup(int stat_group)
+{    
+    config.active_stat_group = stat_group;
```

Comment:
Does this actually preserve the current group across restarts of the client?  I'd think you'd need to also call ConfigSave() in order for the new value to make it to disk.